### PR TITLE
stage 2 task for HNG

### PR DESCRIPTION
**Pull request by EmmyMay**
Making a PR for the stage 2 task

### CODE REVIEW

**Review by Ritika-Agrawal811 — COMMENTED** [review]
add prefix HNG- with your hng ID

**Comment by EmmyMay** [comment]
Bro ple check the instructions. There was no hyphen after the HNG by the mentor

**Comment by Ritika-Agrawal811** [comment]
No I am not talking about that HNG. The prefix in your ID. Check your board HND ID format is "HNG-XXXXX".

File: scripts/myscript.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Hello World, this Emmy Leke with HNGi7 ID 05176 using Javascript for stage 2 task");
```
Comment (ifatoki) [inline scripts/myscript.js]: Please your HNG ID  should be in the form `HNG-02576`

**Review by ifatoki — CHANGES_REQUESTED** [review]
Your filename needs to follow the naming convention [the 5 digit number in your HNG_ID]-[slack-username].[file_ext] for example 01234-itunuloluwa.js

File: scripts/myscript.js (hunk 1 above)
Comment (EmmyMay) [reply scripts/myscript.js]: I have made the fixes.

**Review by McToluene — CHANGES_REQUESTED** [review]
Please add email at the end of your output. Check here https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700

**Comment by EmmyMay** [comment]
Email address has been appended

**Review by McToluene — APPROVED** [review]

**Review by McToluene — APPROVED** [review]

**Review by melissaugrai — APPROVED** [review]

**Review by ifatoki — APPROVED** [review]

**Pull request merged**